lib/applications/resolvers.ts

```diff
@@ -931,6 +931,18 @@ export const updateApplicationPaymentInformation: Resolver<
   const { input } = args;
   const { id, donationAmount, shippingPostalCode, billingPostalCode, ...data } = input;
 
+  const application = await prisma.application.findUnique({ where: { id } });
```

Comment:
We should add a `select` clause here to just select the `paidThroughShopify` field (performance reasons)

Reply:
Nice catch! ✅